lambda/shared/src/main/scala/feral/lambda/IOLambda.scala

```diff
@@ -24,14 +24,15 @@ import cats.syntax.all._
 import io.circe.Decoder
 import io.circe.Encoder
 
-abstract class IOLambda[Setup, Event, Result](
+abstract class IOLambda[Event, Result](
     implicit private[lambda] val decoder: Decoder[Event],
     private[lambda] val encoder: Encoder[Result]
-) extends IOLambdaPlatform[Setup, Event, Result] {
+) extends IOLambdaPlatform[Event, Result] {
 
   protected def runtime: IORuntime = IORuntime.global
 
-  def setup: Resource[IO, Setup]
+  protected type Setup
```

Comment:
Cool, I didn't realize a concrete class/object can just decline to define this and the compiler doesn't complain.